src/lib/git.sh

```diff
@@ -0,0 +1,51 @@
+# The git.sh file provides git easy functionalities for kworkflow code.
+
+# Run git bisect
+#
+# @good: good commit hash
+# @bad: bad commit hash
+# @command: test performed by git bisect
+function run_git_bisect()
+{
+  local good="$1"
+  local bad="$2"
+  local command="$3"
+
+  git bisect start
+  if [[ $? != 0 ]]; then
+    printf '%s\n' 'git bisect start failed'
+    exit 22 # EINVAL
```

Comment:
Avoid exit in a library function since this makes the debug very hard. Since this is a library, we should return an error code and let the caller handle the possible failure.